tests/test_unimpeded.py

```diff
@@ -0,0 +1,17 @@
+from unimpeded.unimpeded import unimpeded
+
+# These locations will be changed to Zenodo links in the future.
+loc = 'local'
+if loc == 'local':
+    location = '/Users/ongdily/Documents/Cambridge/project2/codes/'
+elif loc == 'hpc':
+    location = '/home/dlo26/rds/rds-dirac-dp192-63QXlf5HuFo/dlo26/'
+
+
+def test_get():
```

Comment:
In test_get(), how can I use assert statement to test whether the correct dataset is loaded?